libs/util/git/src/git-client.service.ts

```diff
@@ -101,6 +102,12 @@ export class GitClientService {
     return this.provider.getOrganization();
   }
 
+  async pullPrivatePlugins(
+    pullPrivatePluginsArgs: PullPrivatePluginsArgs
```

Comment:
do we want to expose more granular methods and wrap the process from the external service?

Reply:
I have no strong opinion on that. Feel free to clarify what you meant. 
Since we already have a method for "createPullRequest" here, I thought of following the same architecture. I prefer to keep it as-is for now and, if needed, refactor it later, but I'm open to having another suggestion.
@yuval-hazaz